packages/mobile/src/components/track-list/TrackListItem.tsx

```diff
@@ -266,11 +273,11 @@ const TrackListItemComponent = (props: TrackListItemComponentProps) => {
     return isActive && getPlaying(state)
   })
   const isPlaylistAddable = useIsGatedContentPlaylistAddable(track)
+  const isPlayable = !isLocked || isContentUSDCPurchaseGated(streamConditions)
```

Comment:
maybe overkill, but tempted to say we should either:
A) put the usdc condition in a named boolean called `isPreviewable`
B) add a comment saying why usdc implies playable